### PR TITLE
#266 Use HikariCP and newer versions of postgres and scalikejdbc

### PR DESCRIPTION
**Pull request by gunnarvelle**
GlobalDigitalLibraryio/issues#266
* Better connection pooling.

### CODE REVIEW

File: src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala [hunk 1]
```diff
@@ -84,6 +84,7 @@ object BookApiProperties extends LazyLogging {
   lazy val MetaSchema = prop(PropertyKeys.MetaSchemaKey)
   val MetaInitialConnections = 3
```
Comment (runarfu) [inline src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala]: Blir denne brukt etter denne endringen eller kan den slettes?

Comment (gunnarvelle) [reply src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala]: Fra dok til Hikari:
However, for maximum performance and responsiveness to spike demands, we recommend not setting this value and instead allowing HikariCP to act as a fixed size connection pool.

Comment (runarfu) [reply src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala]: Den kan altså fjernes?

Comment (gunnarvelle) [reply src/main/scala/io/digitallibrary/bookapi/BookApiProperties.scala]: Jepp. Tar den bort.

**Pull request merged**